src/main/resources/application.properties

```diff
@@ -1,5 +1,6 @@
 server.contextPath=
 server.port=8080
+server.max-http-request-header-size=32KB
```

Comment:
Just curious, do you know what the default size is?

Reply:
Default is 8KB - https://docs.spring.io/spring-boot/docs/current/reference/html/application-properties.html#application-properties.server.server.max-http-request-header-size